src/SFA.DAS.EmployerApprenticeshipsService.Infrastructure/Services/ReferenceDataService.cs

```diff
@@ -92,23 +217,46 @@ private List<OrganisationName> FilterOrganisationsByType(IEnumerable<Organisatio
             var result = _cacheProvider.Get<List<OrganisationName>>(cacheKey);
             if (result == null)
             {
-                var orgs = await _client.SearchOrganisations(searchTerm);
+                //var orgs = await _client.SearchOrganisations(searchTerm);
+                var processedSearchTerm = CleanSearchTerm(searchTerm);
+                var orgs = await _client.SearchOrganisations(processedSearchTerm);
 
                 if (orgs != null)
                 {
-                    result = orgs.Select(ConvertToOrganisation).ToList();
+                    var convertedOrgs = orgs.Select(ConvertToOrganisation).ToList();
+
+                    result = SortOrganisations(convertedOrgs, searchTerm);
+
                     _cacheProvider.Set(cacheKey, result, new TimeSpan(0, 15, 0));
                 }
             }
             return result;
         }
 
+        private string CleanSearchTerm(string searchTerm)
+        {
+            if (string.IsNullOrWhiteSpace(searchTerm))
+                return searchTerm;
+
+            foreach (var termToRemove in _termsToRemove)
+            {
+                if (!searchTerm.ToLower().EndsWith(termToRemove.ToLower()))
```

Comment:
Given you're using ToLower() more than once its probably better to store the result and use that multiple times